### PR TITLE
Documentation: future option also works for collections

### PR DESCRIPTION
**Pull request by DirtyF**
Following the discussion on #4676

cc/ @jekyll/documentation

### CODE REVIEW

File: site/_docs/configuration.md [hunk 1]
```diff
@@ -214,7 +214,7 @@ class="flag">flags</code> (specified on the command-line) that control them.
     <tr class="setting">
       <td>
         <p class="name"><strong>Future</strong></p>
-        <p class="description">Publish posts with a future date.</p>
+        <p class="description">Publish posts or any collection with a future date.</p>
```
Comment (parkr) [inline site/_docs/configuration.md]: "or any collection document"?

Comment (DirtyF) [inline site/_docs/configuration.md]: @parkr sorry for my bad english

Comment (parkr) [inline site/_docs/configuration.md]: @DirtyF Not at all! It was fine English as it was, but I nitpick the docs to see if we can be more precise. Thanks for submitting this patch! :heart:

**Comment by parkr** [comment]
@jekyllbot: merge +site

**Pull request merged**